### ISSUE TITLE
XBB.2.3 with ORF1a:L3588F (nsp6_L19F), S:F456L, N:K374Q, ORF1a:P1862S (nsp3_P1044S) and S:Q954Y (6 seqs, USA)

### ISSUE THREAD

**Issue by NkRMnZr** [sars-cov-2-variants/lineage-proposals#721]
**Defining Mutations:** `XBB.2.3` > `T22615C` > `C25096T` >` A23800G` > `C2773T, G11029T(ORF1a:L3588F), T22928C(S:F456L), A29393C(N:K374Q)` > `C5849T(ORF1a:P1862S), C13378T, T21886C, C24422T(S:Q954Y)`
**Query:** `T22615C, T22928C, C24422T`
**Earliest seq:** 2023-06-22 (`EPI_ISL_17955712`, Texas, USA)
**Latest seq:** 2023-08-16 (`EPI_ISL_18144400`, New York, USA)
**Sampled Countries:** USA (8: NY/2; TX/2; UT/1; WY/1)

<details><summary>Genomes: </summary>

`EPI_ISL_17955712, EPI_ISL_18057563, EPI_ISL_18113774, EPI_ISL_18119306, EPI_ISL_18129391, EPI_ISL_18144400`
</details>

**UShER:**
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_3d354_f833d0.json?label=id:node_7374550
![details](https://github.com/sars-cov-2-variants/lineage-proposals/assets/125747944/f02abb23-7df1-4d23-bbd1-709b615f0230)

**Trivia:**
- XBB.2.3 started to get some key mutations like `S:F456L`.
- This lineage has quite a lot of silent mutations on Spike, eg. `T21886C/S:T108`, `T22615C/S:Y351`, `A23800G/S:S746`, `C25096T/S:N1178`, until finally got a 2-nuc mutation `S:H954Y (C24422T, A24424T)`
![954h](https://github.com/sars-cov-2-variants/lineage-proposals/assets/125747944/e6adf42b-0f40-46ae-b339-ee5ee31603c3)

**Comment by FedeGueli** [comment]
too old to track.

**Comment by krosa1910** [comment]
@FedeGueli I would disagree with closing this for its sluggishness and outdatedness, but rather focus on its other features.
First, it reappeared in New York recently with a 3 seqs branch in August, so at least not so bad at growing, and being more recent, but that's not my major point here.
Look at the horizontal coordinate and details about this - Despite being at 9 seqs, it has ran from California to Ohio, Texas, Wisconsin, Utah and finally settling down in New York, that's six states that are not geographically close under any standard. Moreover, when this variants leaped to a new state, it develop some new mutations, so that the newest New York sequences have already completed 4 "mutation steps" and gained 13 additional mutations.
Do not treat this as a tiny branch that would soon fade out in US, but instead think about it as a live textbook example of stepwise mutation. 
<img width="1085" alt="1A80EF31-B16A-4907-BE82-B143FA5EF766" src="https://github.com/sars-cov-2-variants/lineage-proposals/assets/131021067/c6cd052c-a5cc-40b3-bd80-0ef4c0ea795a">

**Comment by FedeGueli** [comment]
Ok reopening for a while. Yes it os def interesting bit accumulation of sequences was extremely low considering it is in a country with the best sequencing at this time. As i think everyone knows my closing are never definitive, sometimes is also to focus on things with growth saving some time tonthe proposer.

**Comment by FedeGueli** [comment]
It cannot compete with Hk.3 : https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?nextcladePangoLineage=HK.3*&nucMutations1=T22615C%2CT22928C%2CC24422T&analysisMode=CompareToBaseline&
